frappe/desk/page/setup_wizard/install_fixtures.py

```diff
@@ -28,4 +31,135 @@ def update_genders_and_salutations():
 				# make sure DuplicateEntryError is for the exact same doc and not a related doc
 				pass
 			else:
-				raise
\ No newline at end of file
+				raise
+
+@frappe.whitelist()
+def make_administrative_area_fixtures():
+    
+    frappe.utils.background_jobs.enqueue(
+        make_administrative_areas,
+        queue="long",
+        timeout=36000,
+        enqueue_after_commit=True,
+    )
+
+def bulid_administrative_area_tree(administrative_areas, country):
+    administrative_states = [area for area in administrative_areas if area['administrative_area_type'] == "state"]
+    lft = 1
+    for state in administrative_states:
+        all_children = [area for area in administrative_areas if area['parent_unique_name'].startswith(state['self_unique_name'])]
+        all_county_children = [area for area in administrative_areas if area['parent_unique_name'] == state['self_unique_name']]
+        rgt = lft + 2*len(all_children) + 1
+        state.update({"lft": lft, "rgt": rgt})
+        if (rgt - lft) == 1:
+            lft = lft + 2
+        else:
```

Comment:
why not just use `rebuild_tree` from NestedSetModel ?

Reply:
@rmehta I have removed the build_administrative_area function which was used to calculate lft and rgt and replaced its functionality by rebuild_tree function from nested set

Another function get_parent_name was written assuming lft, rgt are available beforehand. Now because lft, rgt will be calculated after insertion have modified that function also